### PR TITLE
Lt4.15

### PR DESCRIPTION
**Pull request by davidcarlisle**
## Status of pull request

- Feedback wanted 
- Under development
- Ready to merge

## Checklist of required changes before merge will be approved
- [x] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [x] `ltnewsX.tex` updated (covered by existing entry)

### CODE REVIEW

File: required/tools/longtable.dtx [hunk 1]
```diff
@@ -1282,19 +1282,22 @@
 %    \end{macrocode}
 % \changes{v3.16}{1995/11/09}
 %      {Measure the first line of the table}
+% \changes{v4.15}{2021/04/88}
+%      {silence \cs{vbadness}}
```
Comment (FrankMittelbach) [inline required/tools/longtable.dtx:1286]: gonna be a long month this one :-)

**Review by FrankMittelbach — CHANGES_REQUESTED** [review]

**Comment by FrankMittelbach** [comment]
you may have to merge in development to get rid of test file problems due to the linebreaking issues in the suite at the moment.

**Comment by davidcarlisle** [comment]
Yes I believe the current plan was to wait until you/Phelype had finished
the current round of hook adjustments before merging in these smaller
feature branches, the merge this way just adding the LT update to develop
looks like it should merge cleanly,

**Comment by FrankMittelbach** [comment]
> Yes I believe the current plan was to wait until you/Phelype had finished

didn't put enough commas I guess:-) I meant merge current develp into your branch because I have updated all the tests that fail in develop, that should give you a chance to see that your changes do not generate new diffs

**Review by FrankMittelbach — APPROVED** [review]
in my opinion this is ready for merge

**Pull request merged**